ethereum_bridge/src/storage/mod.rs

```diff
@@ -1,4 +1,6 @@
 //! Functionality for accessing the storage subspace
-pub use namada_core::ledger::eth_bridge::storage::bridge_pool;
+pub mod proof;
 pub mod vote_tallies;
-pub use namada_core::ledger::eth_bridge::storage::{wrapped_erc20s, *};
+pub use namada_core::ledger::eth_bridge::storage::{
+    bridge_pool, wrapped_erc20s, *,
+};
```

Comment:
```suggestion
// TODO: the old location should be deprecated and eventually the modules below should be moved here
pub use namada_core::ledger::eth_bridge::storage::*;
```
nit: we don't need the specific module names if we are using `*`, we should use one or the other?
we should probably have a TODO/issue to eventually move those modules (once all imports using the old path have been updated)

Reply:
we shouldn't even need the star import here. I might make a PR some time later to get rid of it